### ISSUE TITLE
Unable to install LMS on Frappe v15 site

### ISSUE THREAD

**Issue by upcc-jesse** [frappe/lms#1104]
**Describe the bug**
According to #737 I should be able to "use the main branch" of LMS with Frappe v15. There were no specific instructions in that post, so I tried to install it then switch the branch to `main` for LMS - but I can't even get past building LMS. I feel like this is probably just user error on my part, but I don't know how to proceed.

For reference:

```
$bench version
erpnext 15.41.1
frappe 15.47.1
wiki 2.0.0
```

**To Reproduce**

1. I ran `bench get-app lms` which downloaded and tried to build it, but failed. I expected this since I'm supposed to be on the `develop` branch for Frappe and I'm not. Output: https://pastebin.com/MtBwkcnx
2. I then tried `bench switch-to-branch main lms --upgrade` (this was just a guess). Output: https://pastebin.com/fXDriGQ5
3. I then tried `bench build --app lms`. It errored out. Output: https://pastebin.com/Eqy7TqYn

**Expected behavior**
LMS will be compatible with Frappe v15 on the main branch and install without issue.

**Screenshots**

**Additional context**

**Comment by pateljannat** [comment]
@upcc-jesse yes ; this is an issue with your setup. Were you able to get past this?

**Comment by upcc-jesse** [comment]
Yes, it was an issue caused by limited RAM. I was running it on a 2GB virtual instance; upgrading to a 4GB instance allowed the commands to complete successfully.
